### ISSUE TITLE
Exit dialogue

### ISSUE THREAD

**Issue by kineotesting** [cgkineo/adapt-pageIncompletePrompt#18]
On a recent course, the designer included some helpful dialogue for the page incomplete prompt that seems a lot more informative and user friendly than the current default text. Could we adopt this as the default?

You haven’t looked at all the content in the tutorial. 

If you’d like to complete the tutorial, select Stay in the tutorial. Then select the progress bar at the top of the screen…selecting a heading.

If you’d like to leave the tutorial, select **Exit course.**

**Comment by kineotesting** [comment]
This would mean changing the default button names at the bottom of this pop up to **Exit course** and **Stay in the course** as well.

**Comment by moloko** [comment]
@kineotesting I think you have logged this against the wrong extension - should be on [adapt-close](https://github.com/cgkineo/adapt-close)

**Comment by moloko** [comment]
also just to note this does seem to be written in a way that would be more appropriate for a single-page course

**Comment by kineotesting** [comment]
apols Matt - maybe I am confusing two things here - what happens when you navigate away from a page, and what happens when you leave a course entirely - I guess this actually applies more to leaving an incomplete page - and maybe the buttons should be Exit page and Stay on the page (if that's the most generic, understandable terminology for a user)

**Comment by moloko** [comment]
Yep, this extension is for displaying a popup when leaving an incomplete page. 

I certainly like the idea of making the text clearer - in fact it would be great if a designer could review all these pieces of dialog text and ARIA labels - but currently the text above needs a few changes to make it applicable to this extension.

**Comment by kineotesting** [comment]
just been conferring with Pete on this Matt (and have been talking to Em and JCW too, prior - just FYI) - how about this? 
**Are you sure you want to leave?**
You haven’t looked at all the content on the page. 
If you’d like to complete the page, select **Stay on the page**. Then select the progress bar at the top of the screen. This reveals a panel which shows you which parts of the page you still need to complete. Jump straight to the relevant part of the page by selecting a heading.
If you’d like to leave the tutorial, select **Exit page**.
